Dockerfile.dev

```diff
@@ -0,0 +1,22 @@
+FROM registry.fedoraproject.org/fedora:latest
+
+RUN mkdir /app
+WORKDIR /app
+
+# Some dependencies are only needed for aarch64
+RUN dnf install -y bubblewrap skopeo lvm2 rpm-ostree cryptsetup selinux-policy-targeted selinux-policy-devel qemu-img e2fsprogs \
```

Comment:
You are right - this optimization step was never done and this is the original version of the Dockerfile, I'll refine

Reply:
I thought about this a little more and wouldn't do it due to development "setup speed".
This would result in reinstalling all dependencies in the container when you change a bit of code.
But I get your point and will think about other ways to reduce duplication here.